src/math/operations/truncateOps.ts

```diff
@@ -257,11 +259,131 @@ const ambos = makeTruncateTrio({
   },
 })
 
+function getDirection(v1: Vec3D, v2: Vec3D) {
+  return v2.sub(v1).getNormalized()
+}
```

Comment:
This feels like it would be a useful function in other places… let's find where they are